app/models/dialog_tab.rb

```diff
@@ -11,17 +11,15 @@ def to_h
   end
 
   def values_to_h
-    result = {}
     ordered_dialog_resources
-    result
```

Comment:
It sure looks like a bug, and not too surprising it looks like this code is not used anywhere.

The `values_to_h` method is only called from the `to_h` method which does this: `[name.to_sym, values_to_h]`.  The `name` column does not exist so this code would raise an error.

I say you delete both methods `to_h` and `values_to_h`